tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -2052,6 +2053,27 @@ bool CorProfiler::ProfilerAssemblyIsLoadedIntoAppDomain(AppDomainID app_domain_i
            managed_profiler_loaded_app_domains.find(app_domain_id) != managed_profiler_loaded_app_domains.end();
 }
 
+Version CorProfiler::GetProfilerAssemblyVersion(AppDomainID app_domain_id)
```

Comment:
Is this function being called? If not, can we delete it?

Reply:
Done in 0a4342ceeac0232076ad3175ab511c3f7fda851f.